profiler.rst

```diff
@@ -110,16 +110,10 @@ need to create a custom data collector. Instead, use the built-in utilities to
     Consider using a professional profiler such as `Blackfire`_ to measure and
     analyze the execution of your application in detail.
 
-Enabling the Profiler Conditionally
```

Comment:
Please add the following above the headline, to make the old link work again:

.. _enabling-the-profiler-conditionally: